docs/components/DataList/Web.stories.tsx

```diff
@@ -261,8 +300,12 @@ const Template: ComponentStory<typeof DataList> = args => {
     </DataList>
   );
 
-  function handleActionClick(item: (typeof mappedData)[number]) {
-    alert(`You clicked the action for ${item.label}`);
+  function handleActionClick(item: (typeof mappedData)[number] | undefined) {
```

Comment:
protip: question mark should suffice

```suggestion
  function handleActionClick(item?: (typeof mappedData)[number]) {
```